### PR TITLE
Move Foreman/Puppet integration tests to their own file

### PR DESCRIPTION
**Pull request by domcleal**
Trying to cut down on the contents of fb-install-foreman.bats, and lay the way for doing an upgrade task followed by tests (instead of tests being part of installation).

### CODE REVIEW

File: fb-install-foreman.bats [hunk 1]
```diff
@@ -171,93 +171,14 @@ EOF
...
-
-@test "set default root password to 'redhat'" {
-  echo 'Setting["root_pass"] = "$1$redhat$9yxjZID8FYVlQzHGhasqW/"' | foreman-rake console
-}
-
-@test "set idle timeout" {
-  echo 'Setting["idle_timeout"] = 9999' | foreman-rake console
-}
-
-@test "set entries per page" {
```
Comment (domcleal) [inline fb-install-foreman.bats]: I removed these as I couldn't see a use for them, seems like it's user preferences and not required.  I moved the root pass to the libvirt file for provisioning support.

File: README.md [hunk 2]
```diff
@@ -42,10 +42,15 @@ be specified:
 * `FOREMAN_USE_ORGANIZATIONS`: whether to use organizations or not (value can be true/false)
 * `FOREMAN_USE_LOCATIONS`: whether to use locations or not (value can be true/false).
 
-*PuppetLabs Puppet installation test*
+*Foreman Puppet integration test: fb-puppet-tests.bats*
```
Comment (ekohl) [inline README.md]: It's not very consistent to give the exact bats name. I don't mind it, but I'd either do it everywhere or nowhere.

Comment (domcleal) [inline README.md]: Thanks, I did it everywhere in a new commit.

**Comment by ekohl** [comment]
Thanks!

**Pull request merged**